src-tool/Pact/Analyze/Translate.hs

```diff
@@ -1433,6 +1433,12 @@ translateNode astNode = withAstContext astNode $ case astNode of
     tsStaticCapsInScope %= Set.insert capName
     return app
 
+  AST_DiffTime a b -> translateNode a >>= \case
```

Comment:
Tiny-tiny nitpick: I think you might save on nested `case`s and unexpected node handling if you do smth like
```haskell
(translateNode a, translateNode b) >>= \case
  (Some STime a', Some STime b') -> pure ...
  _ -> unexpectedNode astNode
```

Dunno if it's worth it here, so entirely up to you!

Reply:
It would require to use something like `liftM2 (,) (translateNode a) (translateNode b)`, and I'm not sure if this provides any benefit tbh

Reply:
Oh yep, you're totally right, disregard that!